### PR TITLE
Issue #14937: Migrated section 5.2.5 Non Constant Field Names to chapter wise testing

### PR DESCRIPTION
**Pull request by Zopsss**
#14937

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter5naming/rule525nonconstantfieldnames/InputNonConstantNamesBasic.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.google.checkstyle.test.chapter5naming.rule525nonconstantfieldnames;
+
+public class InputNonConstantNamesBasic
+{
+    public int mPublic; // violation 'Member name 'mPublic' must match pattern .*.'
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter5naming/rule525nonconstantfieldnames/InputNonConstantNamesBasic.java:5]: Remove trailing .*

**Review by romani — CHANGES_REQUESTED** [review]
Item

File: src/it/resources/com/google/checkstyle/test/chapter5naming/rule525nonconstantfieldnames/InputNonConstantNamesBasic.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter5naming/rule525nonconstantfieldnames/InputNonConstantNamesBasic.java]: done and conflicts resolved

**Review by romani — APPROVED** [review]
Ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Comment by rdiachenko** [comment]
@Zopsss please resolve conflicts

**Review by romani — APPROVED** [review]

**Pull request merged**